src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -39,6 +43,14 @@ public AddressBook(ReadOnlyAddressBook toBeCopied) {
 
     //// list overwrite operations
 
+    /**
+     * Replaces the contents of the person list with {@code persons}.
+     * {@code persons} must not contain duplicate persons.
```

Comment:
Should be events instead of persons for this method
```suggestion
     * Replaces the contents of the event list with {@code events}.
     * {@code persons} must not contain duplicate events.
```

Reply:
Missing 1 more events

```suggestion
     * Replaces the contents of the event list with {@code events}.
     * {@code events} must not contain duplicate events.
```